kivymd/uix/behaviors/__init__.py

```diff
@@ -7,7 +7,7 @@
 
 # flake8: NOQA
 from .hover_behavior import HoverBehavior  # isort:skip
-from .backgroundcolorbehavior import (
+from .backgroundcolor_behavior import (
```

Comment:
Have you checked to see if this breaks the library and demo application dependencies?